install/install.sh

```diff
@@ -25,14 +25,16 @@ if [ -n "${S3_BUILD_CACHE_KEY}" ]; then
 fi
 spack mirror list
 
-# Add flux, pytest and flake8 to chiltepin spack environment
+# Add flux, pytest, flake8, and black to chiltepin spack environment
 spack add python
 spack add py-pip
 spack add py-pytest
 spack add flux-core@0.53.0
 spack add flux-sched@0.28.0
 spack add py-flake8
 spack add py-pytest-flake8
+spack add py-black
+spack add py-pytest-black
```

Comment:
Spack doesn't have a `py-pytest-black` package.  So, if you want to install `pytest-black` for use as a `pytest` plugin (i.e. `pytest --black ... `), then you'll need to install it with pip.  You can see later in that file where parsl gets installed with pip.  You could do the installation there with `python -m pip install pytest-black` or something like that.